packages/agent/src/canisterStatus/index.ts

```diff
@@ -229,7 +245,12 @@ export const request = async (options: {
   });
 
   // Fetch all values separately, as each option can fail
-  await Promise.all(promises);
+  try {
+    await promises[0];
```

Comment:
should this go in a `for (const promise of promises)...` if you want to fetch each value separately? alternatively, theres `Promise.allSettled`